compiler/emscripten.cmake

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2016, Alexandre Pretyman
+# All rights reserved.
+
+if(DEFINED POLLY_COMPILER_EMSCRIPTEN_CMAKE)
+  return()
+else()
+  set(POLLY_COMPILER_EMSCRIPTEN_CMAKE 1)
+endif()
+
+include(polly_fatal_error)
+
+string(COMPARE EQUAL "$ENV{EMSCRIPTEN}" "" _is_empty)
+if(_is_empty)
+  polly_fatal_error(
+    "EMSCRIPTEN environment variable not set. Set up your Emscripten environment variables with emsdk_env.sh"
```

Comment:
if you are referring to use polly.py instead of the `emsdk_env.sh` script, I put this message here because `emsdk_env.sh` is the script that comes with emscripten to setup the environment variables needed by emscripten